src/Cairo.jl

```diff
@@ -438,6 +438,13 @@ function stroke_preserve(ctx::CairoContext)
     restore(ctx)
 end
 
+function paint_with_alpha(ctx::CairoContext, a = 0.5)
```

Comment:
How useful is the `a=0.5` default? Maybe it would be better not to provide any default?